catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/metacard/AbstractBasicMetacardUpdater.java

```diff
@@ -28,7 +30,7 @@ protected AbstractBasicMetacardUpdater(String attributeName) {
     }
 
     @Override
-    public final void update(Metacard parent, Metacard child) {
+    public final void update(Metacard parent, Metacard child, Context context) {
```

Comment:
I was taught a long time ago that when providing an implementation for a method in an abstract class to always make it final. That way classes that extend the abstract class are forced to always have the same behavior.